autoscheduler/frontend/src/components/LandingPage/About/About.tsx

```diff
@@ -0,0 +1,151 @@
+            {contributor.linkedInURL ? (
+              <IconButton
+                className={styles.overrideIconButton}
+                onClick={(): void => { window.open(contributor.linkedInURL); }}
+              >
+                <LinkedIn />
+              </IconButton>
+            ) : null}
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  const contributorsJSX: JSX.Element[] = contributors.map(
+    (contributor) => getContributorRow(contributor),
+  );
+  const pastContributorsJSX: JSX.Element[] = pastContributors.map(
+    (contributor) => getContributorRow(contributor),
+  );
+
+  const dialog: JSX.Element = (
+    <Dialog
+      open={open}
+      onClose={(): void => setOpen(false)}
+    >
+      <DialogTitle disableTypography>
+        <div className={styles.dialogTitle}>
+          <Typography variant="h6">
+            About
+          </Typography>
+          <ThemeProvider theme={whiteButtonTheme}>
+            <IconButton color="primary" onClick={(): void => setOpen(false)}>
+              <CloseIcon />
+            </IconButton>
+          </ThemeProvider>
+        </div>
+      </DialogTitle>
+      <DialogContent className={styles.dialogContent}>
+        <Typography>
+          Rev Registration is an open-source&nbsp;
+          <a href="https://aggiecodingclub.com">Aggie Coding Club</a>
+          &nbsp;project led by Gannon Prudhomme and Ryan Conn.
+        </Typography>
+        <Typography>
+          You can find all of our code on our&nbsp;
+          <a href="https://github.com/aggie-coding-club/Rev-Registration">GitHub repository</a>
+          .
+        </Typography>
+        <div>
+          <Typography variant="h6">
+            Contributors
+          </Typography>
+          {contributorsJSX}
+        </div>
+        <div>
+          <Typography variant="h6">
+            Past Contributors
+          </Typography>
+          {pastContributorsJSX}
+        </div>
+        <div>
+          <Typography variant="h6">
+            License
+          </Typography>
+          <Typography>
+            Rev Registration is under the&nbsp;
+            <a href="https://github.com/aggie-coding-club/Rev-Registration/blob/master/LICENSE">
+              GNU GPL v3 License
+            </a>
+            .
+          </Typography>
+        </div>
+        <Typography>
+          Questions? Suggestions? Contact us at&nbsp;
+          <a href="mailto:register.rev@gmail.com">
+            register.rev@gmail.com
+          </a>
+          .
+        </Typography>
+      </DialogContent>
+    </Dialog>
+  );
+
+  // The link on the home page to open the dialog
+  const link: JSX.Element = (
+    <div className={styles.dialogLink}>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+      <Link
+        component="button"
+        variant="body2"
+        onClick={(): void => {
+          setOpen(true);
+        }}
+        style={{
+          color: 'gray !important',
+        }}
```

Comment:
I'm not seeing this actually affect anything, isn't it supposed to style the link text? Regardless I believe we said we were going to make it maroon so this can be removed

Reply:
Yeah was just there by mistake, forgot to remove since it didn't affect anything